### DISCUSSION TITLE
Public IP on console login screen

### DISCUSSION THREAD

**Discussion by ghenry** [telekom-security/tpotce#1026] [Q&A]
Hi all,

I noticed on the console login welcome screen your public IP address is shown. What figures that out?

Thanks.

**Comment by t3chn0m4g3** [comment]
It basically serves convenience purposes as it tells you the public IP address the T-Pot is probably reachable at if you desire to test its functionality over the internet.

**Comment by ghenry** [comment]
Thanks. Yeah, but I was wondering what code figured that out?

**Reply by t3chn0m4g3** [reply]
You find the script [here](https://github.com/telekom-security/tpotce/blob/master/bin/myip.sh).

**Comment by ghenry** [comment]
Ah, perfect. Thanks. I thought it would be something like that.
